README.md

```diff
@@ -34,47 +34,39 @@ itself in a few ways:
     their dependencies. How you install different versions of commands is
     entirely up to you.
 
-## Installation
+## Install
 
-1.  Install the `alt` binary.
+### Debian / Ubuntu / Anything using DEBs
 
-    ```sh
-    curl -sL https://github.com/dotboris/alt/raw/master/install.sh | bash -s
-    ```
-
-1.  Add the `alt` shims directory to the top of your `PATH`.
-    This lets `alt` change command versions.
-
-    For BASH:
+1.  Open the [latest release page on Github][latest-release]
+1.  Download the `.deb` file matching your system architecture
+1.  Install the `.deb` file by double clicking on it
+1.  OR Install the `.deb` file from the command line
 
     ```sh
-    echo 'export PATH="$HOME/.local/alt/shims:$PATH"' >> ~/.bashrc
-    export PATH="$HOME/.local/alt/shims:$PATH"
+    sudo apt install ./path/to/alt.deb
     ```
 
-    For ZSH:
+You will probably need to log out & log back in to your desktop session for
+`alt` to be configured.
 
-    ```sh
-    echo 'export PATH="$HOME/.local/alt/shims:$PATH"' >> ~/.zshrc
-    export PATH="$HOME/.local/alt/shims:$PATH"
-    ```
+### OSX (Homebrew & Linuxbrew)
```

Comment:
Shouldn't this be above the `.deb` instructions? And isn't that also available on most Linux distros? The title says `OSX` specifically.

```suggestion
### Brew (Homebrew & Linuxbrew)
```

Reply:
@binaryannie I don't think that it's a good idea.

I think that it's fair to assume that OSX users are going to have homebrew installed if they're interested in `alt`. 

On the other hand, I don't expect many people to have linuxbrew on their linux installation. I don't think that this is the easiest way to install this on linux as you would first have to install linuxbrew.

With the way I have it now:

- OSX users see their operating system. Dead giveaway that this is for them.
- Debian & Ubuntu users see their operating system. Again, dead giveaway.
- Linuxbrew users see the Linuxbrew keyword and know what to use